dask_jobqueue/slurm.py

```diff
@@ -54,27 +51,24 @@ def __init__(self,
         project : str
             Accounting string associated with each worker job. Passed to
             `#SBATCH -A` option.
-        threads_per_worker : int
-            Number of threads per process.
         processes : int
             Number of processes per node.
         memory : str
             Bytes of memory that the worker can use. This should be a string
             like "7GB" that can be interpretted both by PBS and Dask.
         walltime : str
             Walltime for each worker job.
-        interface : str
-            Network interface like 'eth0' or 'ib0'.
-        death_timeout : float
-            Seconds to wait for a scheduler before closing workers
-        extra : str
-            Additional arguments to pass to `dask-worker`
         kwargs : dict
-            Additional keyword arguments to pass to `LocalCluster`
+            Additional keyword arguments to pass to `JobQueueCluster` and `LocalCluster`
         """
-        self._template = """
-#!/bin/bash
 
+        super(SLURMCluster, self).__init__(name=name, processes=processes, **kwargs)
+
+        #TODO has this been tested? This seems weird to use only processes, and not processes * threads
+        # There are no memory limit given to Slurm either?
```

Comment:
I think the initial implementation left those out but we should provide both. @jedwards4b - do you have a comment here?